src/content/language/methods.md

```diff
@@ -44,29 +44,30 @@ class Point {
 Operators are instance methods with special names.
 Dart allows you to define operators with the following names:
 
-|      |      |      |       |
-|------|------|------|-------|
-| `<`  | `+`  | `\|` | `>>>` |
-| `>`  | `/`  | `^`  | `[]`  |
-| `<=` | `~/` | `&`  | `[]=` |
-| `>=` | `*`  | `<<` | `~`   |
-| `-`  | `%`  | `>>` | `==`  |
+|       |      |      |      |       |      |
+|-------|------|------|------|-------|------|
+| `<`   | `>`  | `<=` | `>=` | `==`  | `~`  |
+| `-`   | `+`  | `/`  | `~/` | `*`   | `%`  |
+| `\|`  | `ˆ`  | `&`  | `<<` | `>>>` | `>>` |
+| `[]=` | `[]` |      |      |       |      |
 
 {:.table}
 
 :::note
 You may have noticed that some [operators][], like `!=`, aren't in
-the list of names. That's because they're just syntactic sugar. For example,
-the expression `e1 != e2` is syntactic sugar for `!(e1 == e2)`.
+the list of names. That's because they're just [syntactic sugar][sugar].
+For example, the expression `e1 != e2` is syntactic sugar for `!(e1 == e2)`.
 :::
 
+[sugar]: https://en.wikipedia.org/wiki/Syntactic_sugar
+
 {%- comment %}
   Internal note from https://github.com/dart-lang/site-www/pull/2691#discussion_r506184100:
   -  `??`, `&&` and `||` are excluded because they are lazy / short-circuiting operators
   - `!` is probably excluded for historical reasons
 {% endcomment %}
 
-An operator declaration is identified using the built-in identifier `operator`.
+To identify an operator declaration, use the built-in identifier `operator`.
```

Comment:
I'm not sure what "identify" is doing in this sentence. Maybe: "To declare an operator, use the built-in identifier `operator` followed by the operator you are defining."

Reply:
+1